CompanionApp/CompanionApp.xcodeproj/project.pbxproj

```diff
@@ -443,8 +443,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/hmrc/ios-components";
 			requirement = {
-				kind = exactVersion;
-				version = 5.2.0;
+				kind = upToNextMajorVersion;
```

Comment:
We've chosen to go with exactVersion for all other dependence so I guess we should do the same with our own. What do you think @Testablekarma ?

Reply:
If it was a third party dependency I'd agree but I don't think this would matter too much really as it'll be changing pretty often to go with whatever the latest version is on the comp lib.

Might be worth it for clarity tbf though

Reply:
Updated in ad1c1df

Reply:
👍